commons/commands.py

```diff
@@ -574,7 +574,7 @@
 CLSTR_START_CMD = "cd {}; ./start-cortx-cloud.sh"
 CLSTR_STOP_CMD = "cd {}; ./shutdown-cortx-cloud.sh"
 CLSTR_STATUS_CMD = "cd {}; ./status-cortx-cloud.sh"
-CLSTR_LOGS_CMD = "cd {}; ./logs-cortx-cloud.sh"
+CLSTR_LOGS_CMD = "cd {}; ./logs-cortx-cloud.sh -s solution.yaml --all True"
```

Comment:
@ravindrachoudhari  @sonalk0209 @rshenoy0831 
This change will impact all tests which collect support bundle.